### PR TITLE
update price drop docs to look like back in stock

### PR DESCRIPTION
**Pull request by mkigikm**
### Summary

Updates price drop docs to look like back in stock. In particular, added a example custom event for subscriptions.

### Related PRs, issues, or features (optional)

- N/A

### Feature release date (optional)

- N/A

### Contributor checklist

- [ ] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

**Review by shihanxiong — APPROVED** [review]

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs/price_drop_notifications.md [hunk 1]
```diff
@@ -66,4 +87,11 @@ Using {%raw%}``{{canvas_entry_properties.${catalog_update}.item_id}}``{%endraw%}
 
 Use this Liquid tag {%raw%}``{% catalog_items <name_of_your_catalog> {{canvas_entry_properties.${catalog_update}.item_id}} %}}``{%endraw%} at the top of your message, then use {%raw%}`{{items[0].<field_name>}}`{%endraw%} to access data about that item throughout the message.
 
-[1]: {% image_buster /assets/img/price_drop_notifications.png %} 
\ No newline at end of file
+## Considerations
+
+- Users are subscribed for 90 days. If an item does not drop in price in 90 days, the user is removed from the subscription.
+- When using the **Notify all subscribed users** notification rule, Braze will notify 100,000 over 10 minutes.
+- Braze will process up to 10 item updates per minute. This means if you update 11 items in one minute, only the first 10 items can trigger a price drop the notification.
```
Comment (davehensley) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs/price_drop_notifications.md:94]: Nit
```suggestion
- When using the **Notify all subscribed users** notification rule, Braze will notify 100,000 over 10 minutes.
- Braze will process up to 10 item updates per minute. This means if you update 11 items in one minute, only the first 10 items can trigger a price drop notification.
```

**Review by davehensley — APPROVED** [review]

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs/price_drop_notifications.md [hunk 2]
```diff
@@ -66,4 +87,11 @@ Using {%raw%}``{{canvas_entry_properties.${catalog_update}.item_id}}``{%endraw%}
 
 Use this Liquid tag {%raw%}``{% catalog_items <name_of_your_catalog> {{canvas_entry_properties.${catalog_update}.item_id}} %}}``{%endraw%} at the top of your message, then use {%raw%}`{{items[0].<field_name>}}`{%endraw%} to access data about that item throughout the message.
 
-[1]: {% image_buster /assets/img/price_drop_notifications.png %} 
\ No newline at end of file
+## Considerations
+
+- Users are subscribed for 90 days. If an item does not drop in price in 90 days, the user is removed from the subscription.
+- When using the **Notify all subscribed users** notification rule, Braze will notify 100,000 over 10 minutes.
```
Comment (lydia-xie) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs/price_drop_notifications.md:93]:
```suggestion
- When using the **Notify all subscribed users** notification rule, Braze will notify 100,000 users over 10 minutes.
```

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs/price_drop_notifications.md [hunk 3]
```diff
@@ -25,18 +31,33 @@ Follow these steps to set up price drop notifications in a specific catalog.
...
+                "properties": {
+                    "id": "shirt-xl",
+                    "catalog_name": "on_sale_products",
+                    "type": ["price_drop"]
+                }
+            }
+        ]
+    }
+    ```
     <br><br>
```
Comment (lydia-xie) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs/price_drop_notifications.md:56]:
```suggestion
    <br>
```

**Review by lydia-xie — APPROVED** [review]
lgtm!

**Pull request merged**